### PR TITLE
add doc

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

**Review by Xiaochao8 — APPROVED** [review]

**Comment by huihuiw01** [comment]
Please link the PR to corresponding issue.

File: README.md [hunk 1]
```diff
@@ -1 +1,60 @@
-#spring-boot-remote-git-autoconfigure
+spring-boot-remote-git-autoconfigure
+=======================================
+It can download the Spring Boot configurations from remote git repo and load the configuration to Spring boot APP
+
+## Prerequisites
+You should install the git client successfully and the configure the ssh auto verify with remote git repo successfully. It mean that you can use the OS git command pull the remote configuration repo item successfully.
+
+## Add the dependency jar
```
Comment (huihuiw01) [inline README.md:8]: Take Singleton service and client's readme as reference and from users' perspective, is it better to add a parent title 'How to build and use it'? 

And in this part add content about how to build it.

File: README.md [hunk 2]
```diff
@@ -1 +1,60 @@
...
+</dependency>
+
+```
+  gradle:
+```
+implementation 'com.vmware.singleton:spring-boot-remote-git-autoconfigure:{version}'
+
+```
+
+## create the remote git config repository for Spring Boot apps
```
Comment (huihuiw01) [inline README.md:24]: Is it better move it to 'Prerequisites' part?

Comment (tigershi) [reply README.md:24]: It is part of how to use it.

File: README.md (hunk 1 above)
Comment (tigershi) [reply README.md:8]: I tend to think that 'how to build' belong to how to contribute part. It only introduces 'how to use it'

Comment (huihuiw01) [reply README.md:8]: That's ok. My main point is 'how to use it' contains below 4 steps:
## Add the dependency jar 
## create the remote git config repository for Spring Boot apps
## Add the configuration item to application configuration(application.properties, application.yml, application.yaml)
## Start the spring boot app
right? Make each step as first level heading looks ungraceful, so had better add 'how to use it' as first level heading which parallel with 'Prerequisites', and change them as secondary heading under 'how to use it', please refer to https://github.com/vmware/singleton/tree/g11n-java-client readme.
how do you think?

Besides, will you add contribution file in this PR?

**Review by huihuiw01 — APPROVED** [review]

**Review by Xiaochao8 — APPROVED** [review]

**Pull request merged**